api/impl/cactusBlock.c

```diff
@@ -110,8 +110,9 @@ Segment *block_getInstanceP(Block *block, Segment *connectedSegment) {
 }
 
 Segment *block_getInstance(Block *block, Name name) {
```

Comment:
These *_getStaticNameWrapper removals were to make everything thread-safe. (The tree-building is done using multiple threads if enabled.)